db/migrate/20160830233154_add_port_ranges_to_load_balancer_listener.rb

```diff
@@ -0,0 +1,8 @@
+class AddPortRangesToLoadBalancerListener < ActiveRecord::Migration[5.0]
+  def change
+    change_table :load_balancer_listeners do |t|
+      t.column :load_balancer_port_range, :int4range, :default => nil, :null => true
+      t.column :instance_port_range, :int4range, :default => nil, :null => true
```

Comment:
Well, alternatively we could create two separate integer columns for min/max, and achieve something similar.  However if we ever want to see if two port ranges overlap, the queries get pretty nasty; using a range type in postgres means the query language itself is aware of how to perform these lookups, which not only benefits speed but also makes the actual implementation more readable.  But that's my opinion.

I don't know of any downsides (but I haven't used numranges extensively) other than the complexity in having to use Range objects in the AR model. It does tie ManageIQ closer to postgres, but I believe there is already postgres-specific stuff in there (like the jsonb datatypes).

Will accept whichever approach you prefer; what do you think?